packages/presets/src/ai/chat-panel/actions/actions-handle.ts

```diff
@@ -97,18 +96,29 @@ export const PageEditorActions = [
   ...CommonActions,
   {
     icon: CreateIcon,
-    title: 'Create as a page',
+    title: 'Create as a doc',
     handler: (host: EditorHost, content: string) => {
       reportResponse('result:add-page');
       const newDoc = host.doc.collection.createDoc();
       newDoc.load();
       const rootId = newDoc.addBlock('affine:page');
       newDoc.addBlock('affine:surface', {}, rootId);
       const noteId = newDoc.addBlock('affine:note', {}, rootId);
-      newDoc.addBlock('affine:paragraph', { text: new Text(content) }, noteId);
+
       host.spec.getService('affine:page').slots.docLinkClicked.emit({
         docId: newDoc.id,
       });
+      let complete = false;
+      (function addContent() {
+        if (complete) return;
+        const newHost = document.querySelector('editor-host');
```

Comment:
Not considering the situation of multiple hosts, why not `await host.updateComplete`

Reply:
This operation is only related to model. I guess there is no need to `await host.updateComplete`.
As for multiple hosts,  any advices?